timeboost-sequencer/src/sort.rs

```diff
@@ -37,10 +37,14 @@ impl Sorter {
                         }
                         match Transaction::decode(&t) {
                             Ok(tx) => {
+                                let Ok(signer) = tx.tx().recover_signer() else {
```

Comment:
It seems that transactions where signer recovery fails are now dropped, whereas before only the comparison would be affected. Requiring signer recovery is stricter and more straightforward, however would it not make more sense to reject transactions *on submission* when the recovery fails? Why even including them in candidate lists?

Reply:
I guess encrypted transactions get in the way of that approach.